src/lwc/gePaymentGatewayManagement/gePaymentGatewayManagement.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2020 Salesforce.org
+ *     All rights reserved.
+ *
+ *     Redistribution and use in source and binary forms, with or without
+ *     modification, are permitted provided that the following conditions are met:
+ *
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of Salesforce.org nor the names of
+ *       its contributors may be used to endorse or promote products derived
+ *       from this software without specific prior written permission.
+ *
+ *     THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ *     "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ *     LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ *     FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *     COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *     INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ *     BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ *     LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ *     CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ *     LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *     ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *     POSSIBILITY OF SUCH DAMAGE.
+ */
+import { LightningElement } from 'lwc';
+import { buildErrorMessage } from 'c/utilTemplateBuilder';
+import { isEmpty } from 'c/utilCommon';
+import messageLoading from '@salesforce/label/c.labelMessageLoading';
+import setGatewayId from '@salesforce/apex/PS_GatewayManagement.setGatewayId';
+import getGatewayIdFromConfig from '@salesforce/apex/PS_GatewayManagement.getGatewayIdFromConfig';
+
+export default class GePaymentGatewayManagement extends LightningElement {
+
+    showSpinner;
+    gatewayId;
+
+    isReadOnly = true;
+    errorMessage;
+
+    CUSTOM_LABELS = { messageLoading };
+
+    connectedCallback() {
+        this.getGatewayId();
+    }
+
+    _isSuccess;
+    get isSuccess() {
+        return this._isSuccess;
+    }
+    set isSuccess(value) {
+        this._isSuccess = value;
+
+        if (value) { this.isError = false; }
+    }
+
+    _isError;
+    get isError() {
+        return this._isError;
+    }
+    set isError(value) {
+        this._isError = value;
+
+        if (value) { this.isSuccess = false; }
+    }
```

Comment:
Could isError be a simple getter that analyzes the errorMessage property instead of storing both?  

```javascript
get isError() {
        return !!this.errorMessage;
    }
```

Reply:
@kenlewis-sfdo I thought of going that route but it would've required a little more robust solution both in this component and the alert banner component that I felt wasn't necessary with this being a simpler temporary feature in general that will only ever have these two alert banner states.

Reply:
> This is looking really good, Daniel. I left one comment re: whether the _isError prop is needed, and I had one question re: the second A/C on the WI. Should customers be prevented from saving a new Gateway Id when they do not already have one configured (i.e., they are not yet an Elevate customer)?

Since we are keeping this feature simple overall we are just letting the admin override the current gateway id with whatever they enter into the field. We aren't doing any additional checks or validations other than not allowing them to save a blank value.